src/components/layout/nav/search/Search.scss

```diff
@@ -0,0 +1,181 @@
+@import "../../../common/style.scss";
+
+:root {
+  font-size: 16px;
+}
+
+.searchBox {
+  width: 100%;
+  background-color: white;
+  margin: auto;
+  height: 100px;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 0;
+
+  span {
+    position: absolute;
+    padding: 5px 15px;
+    background-color: #e2909066;
+    border-radius: 5px;
+    bottom: 5px;
+    max-width: 400px;
+    color: #464444;
+    font-size: 1.4vw;
+  }
+
+  .searchBar {
+    height: 55%;
+    background-color: #ebebeb;
+    border: 1px solid #d7d7d7;
+    color: #333;
+    position: relative;
+    align-items: center;
+    display: flex;
+    padding-left: 0;
+    padding-right: 0;
+    border-radius: 0.3rem;
+    transition: all 0.3s ease-in-out;
+
+    .searchButtonWrapper,
+    .searchInputWrapper {
+      height: 100%;
+      position: relative;
+      display: inline-block;
+      padding: 0;
+    }
+
+    .searchInput {
+      width: 100%;
+      height: 100%;
+      background-color: #ebebeb;
+      transition: background-color 0.3s ease-in-out;
+      border-top-right-radius: 0 !important;
+      border-bottom-right-radius: 0 !important;
+
+      // To override default styling of Bootstrap FormInput.
+      font-size: 100%;
+
+      &:hover {
+        background-color: white;
+      }
+    }
+
+    button {
+      padding: 0;
+      height: 100%;
+      width: 100%;
+      background-size: 150%;
+      background-color: #8bc6ec;
```

Comment:
i believe this is the same color as burak. move it into common, use that variable here